x-pack/plugins/siem/public/resolver/view/edge_line.tsx

```diff
@@ -65,16 +97,37 @@ const EdgeLineComponent = React.memo(
        */
       transform: `translateY(-50%) rotateZ(${angle(screenStart, screenEnd)}rad)`,
     };
-    return <div role="presentation" className={className} style={style} />;
+
+    let elapsedTimeLeftPosPct = 50;
+
+    if (magFactorX < 1) {
+      const fractionalOffset = (1 / magFactorX) * ((1 - magFactorX) * 10);
```

Comment:
could you explain this calculation?

Reply:
It's just a scaling function that at our current `magFactorX` min of .5 will keep the elapsedTime section centered at a max `fractionalOffset` of 5, and scale back down to 50 as you zoom back in. I just noticed that the elapsedTime div became more off centered as I zoomed out fully, so added this to recenter it as you zoomed out. Open for suggestions on a better / cleaner way! Will also add comments so it's a bit clearer